pyfar/io/io.py

```diff
@@ -962,3 +963,121 @@ def _read_comsol_get_headerline(filename):
     is_complex = 'i' in line
     delimiter = ',' if ',' in line else None
     return header, is_complex, delimiter
+
+
+def read_ita(filename):
+    """
+    Read an `*.ita` file created using the ITA-toolbox.
+
+    ITA-Tollbox is a MATLAB Toolbox for Acoustics [#]_.
+
+    Parameters
+    ----------
+    filename : str
+        The filename. The file extension must be `*.ita`.
+
+    Returns
+    -------
+    data : pyfar.Signal, pyfar.TimeData, pyfar.FrequencyData
+        The audio data contained in the `*.ita` file.
+    object_coordinates : pyfar.Coordinates
+        The object coordinates of the `*.ita` file.
+    channel_coordinates : pyfar.Coordinates
+        The channel coordinates of the `*.ita` file.
```

Comment:
In case we follow sikerstens suggestion below, this should become
```suggestion
    object_coordinates : pyfar.Coordinates, None
        The object coordinates of the `*.ita` file if they exist, ``None`` otherwise.
    channel_coordinates : pyfar.Coordinates, None
        The channel coordinates of the `*.ita` file if they exist, ``None`` otherwise.
```

Reply:
Maybe add a note on the difference of object and channel coordinates? Do we need both? Just asking because I'm not familiar with this.